wagtail/admin/tests/tests.py

```diff
@@ -13,6 +13,7 @@
 from wagtail.admin.auth import user_has_any_page_permission
 from wagtail.admin.mail import send_mail
 from wagtail.admin.menu import MenuItem
+from wagtail.admin.views.tags import WAGTAIL_TAGS_AUTOCOMPLETE_LIMIT
```

Comment:
@aayushman-singh  I see what you were intending here but this change will not actually test that there are ten items but just that the import is used to reflect the item limit.

Which means if there's a typo in this value or something else wrong, it will not throw an error in the test.

Instead, we should leave this test as is and leave the 'internal implementation' of that module outside of the tests.

If there's a future where this gets converted into a setting, then we would create new tests for this.